lib/voyage/templates/api_base_controller.rb

```diff
@@ -0,0 +1,38 @@
+module Api
+  class BaseApiController < ApplicationController
+    # Disable CSRF protection for API calls
+    protect_from_forgery with: :null_session
+
+    # Disable cookie usage
+    before_action :destroy_session
```

Comment:
Not entirely necessary, but this will prevent any attempted cookies on API requests.